Example/Podfile.lock

```diff
@@ -95,16 +95,15 @@ PODS:
   - SDWebImage (3.7.5):
     - SDWebImage/Core (= 3.7.5)
   - SDWebImage/Core (3.7.5)
-  - Sentry (3.9.1):
-    - Sentry/Core (= 3.9.1)
-    - Sentry/KSCrash (= 3.9.1)
-  - Sentry/Core (3.9.1)
-  - Sentry/KSCrash (3.9.1):
-    - KSCrash/Core (~> 1.15.12)
-  - SentryReactNative (0.30.2):
+  - Sentry (3.1.3):
+    - Sentry/Core (= 3.1.3)
+  - Sentry/Core (3.1.3)
+  - Sentry/KSCrash (3.1.3):
+    - KSCrash/Core (~> 1.15.11)
+  - SentryReactNative (0.14.5):
     - React
-    - Sentry (~> 3.9.0)
-    - Sentry/KSCrash (~> 3.9.0)
+    - Sentry (~> 3.1.2)
+    - Sentry/KSCrash (~> 3.1.2)
```

Comment:
Note that after doing a `git pull` on `master`, I had to run `cd Examples && pod update` per the prompt.

Reply:
It looks more like these downgraded :| What exactly did the prompt say?

Reply:
That I needed to run `bundle exec pod update Sentry/KSCrash`